src/validation.js

```diff
@@ -0,0 +1,109 @@
+import * as patterns from './patterns';
+
+/**
+ * Check wherther given value was character or not
+ * @param {*} value given value wanted to cheek it
+ * @returns {boolean} true if was character, otherwise false
+ */
+function isCharacter (value) {
+    return patterns.character.test(value.toString().toLowerCase());
+}
+
+/**
+ * Check wherther given value was valid email or not
+ * @param {*} value given value wanted to cheek it
+ * @returns {boolean} true if was valid email, otherwise false
+ */
+function isValidEmail (value) {
+    return patterns.email.test(value.toString().toLowerCase());
+}
+
+/**
+ * Check whether given both values was equal or not
+ * @param  {*}  firstValue  first value wanted to check
+ * @param  {*}  secondValue second value wanted to check
+ * @returns {boolean} true if was equal, otherwise false
+ */
+function isEqual (firstValue, secondValue) {
+  return firstValue.toString().trim().toLowerCase() === secondValue.toString().trim().toLowerCase();
+}
+
+/**
+ * Check whether given value has been exced given length
+ * @param  {*} value  value want to check
+ * @param  {number} length maximum length value
+ * @returns {boolean} true if was exceed, otherwise false
+ */
+function exceedMaxLength (value, length) {
+    return value.toString().trim().length > length;
+}
+
+/**
+ * Check whether given value has been match minmum length
+ * @param  {*} value  value want to check
+ * @param  {number} length minmum length value
+ * @returns {boolean} true if was match, otherwise false
+ */
+function matchMinLength (value, length) => {
```

Comment:
you can not use arrow syntax with **function**  keyword

Reply:
yes, copy past mistake